data-prepper-plugins/grok-prepper/build.gradle

```diff
@@ -0,0 +1,37 @@
+/*
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  The OpenSearch Contributors require contributions made to
+ *  this file be licensed under the Apache-2.0 license or a
+ *  compatible open source license.
+ *
+ *  Modifications Copyright OpenSearch Contributors. See
+ *  GitHub history for details.
+ */
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation "com.fasterxml.jackson.core:jackson-databind:2.12.4"
```

Comment:
Don't we have this already defined higher up in the hierarchy?

Reply:
We actually don't. Could add this as a dependency to data-prepper-plugins but right now each individual plugin adds this if it is used.

Reply:
It is not a dependency, but you can inherit the version number. So, you should remove the version number from this line.

See: https://github.com/opensearch-project/data-prepper/blob/main/build.gradle#L74